calico-enterprise/operations/monitor/metrics/recommended-metrics.mdx

```diff
@@ -0,0 +1,514 @@
+| Threshold breach recommendations   | Check Felix logs and resource usage.                         |
+| Priority level                     | Recommended.                                                 |
+
+### 99 percentile client ping latency
+
+| 99 Percentile client ping latency |                                                                                                                                                                                                                                                                 |   |   |   |
+|-----------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---|---|---|
+| Metric                            | max by (instance) (typha_ping_latency{quantile="0.99"})                                                                                                                                                                                                         |   |   |   |
+| Explanation                       | This metric tracks the round-trip time from Typha to a client. How long it takes for Typha's clients to respond to pings over the Typha protocol.                                                                                                               |   |   |   |
+| Threshold value recommendation    | An increase in this metric above 1 second indicates that the clients, network, or Typha are more heavily loaded. It is normal for intermittent spikes. Persistent latency above 1 second warrants investigation.                                                 |   |   |   |
+| Threshold breach symptoms         | Typha clients could be behind in time on updates Typha is sending.  Potential symptoms include Calico Network Policies being out-of-sync                                                                                                                        |   |   |   |
+| Threshold breach recommendations  | Check Typha and Felix logs and resource usage.   It is recommended to focus on Felix logs and resource usage first, as there is generally more overhead with Felix and thus more of a chance of overload. Rollout restart Typha(s) and Calico-node(s) if needed |   |   |   |
+| Priority level                    | Recommended                                                                                                                                                                                                                                                     |   |   |   |
+
+### 99 percentile client ping latency
+
+| 99 Percentile client ping latency |                                                              |
+| --------------------------------- | ------------------------------------------------------------ |
```

Comment:
Table is malformed in the deploy preview

Reply:
Fixed.